patdel0/SAGP
# Chore: set up the project to use cloud JSON storage

# Description

This PR sets up a way to interact with our [storage provider](https://getpantry.cloud/) and adds Vite to help protect the production storage by bundling files in `dist/` with the right environment variable values.

Contributors won’t have access to the production storage, a testing environment wasn't created and mocking isn’t included, but I’ve added instructions to help developers set up a realistic testing environment for themselves.

I’ve kept the data structure and implementation of the Coffee Chat user list simple and incomplete by design. The idea is to leave room for others to jump in, contribute, and gain hands-on experience as part of the process.

## Type of change


# How to Test

1. Follow the updated instructions in [ `CONTRIBUTING.md`](https://github.com/patdel0/SAGP/blob/f9718551aac5b059ff0a4a8e3b251382642495cb/docs/CONTRIBUTING.md#setting-up-the-development-environment)
2. `npm run dev`
3. Open the site and confirm you're able to add users and see a list of the seeded users

# Checklist:
